en-US/GetStarted/Commercialize/BuildRetailImage.htm

```diff
@@ -9,13 +9,13 @@
 
 <header>
     <h1  class="c-subheading-1">Build your retail image</h1>
-    <p class="c-paragraph-3">With the customizations you've added, it's time to put them together and test them in a retail build.</p>
+    <p class="c-paragraph-3">With the customizations you've added, you can now put them together and test them in a retail build.</p>
 </header>
 
 <section class="pad-thin">
-    <h3 class="c-subheading-3">1. Build and create the image.</h3>
```

Comment:
Maybe use ordered lists instead of hard coding the index.

Reply:
The indentation gets a little funky :( Let's hold on this for now unless you think it's a blocker.